userspace/chisel/chisel_api.cpp

```diff
@@ -843,7 +843,7 @@ int lua_cbacks::get_thread_table_int(lua_State *ls, bool include_fds, bool bareb
 			lua_pushnumber(ls, (uint32_t)tinfo.m_group.gid);
 			lua_settable(ls, -3);
 			lua_pushliteral(ls, "nchilds");
-			lua_pushnumber(ls, (uint32_t)tinfo.m_nchilds);
+			lua_pushnumber(ls, (uint32_t)0);
```

Comment:
Can we use tinfo.get_num_not_leader_threads() here?

Reply:
sure great catch!